pkg/plugins/plugins.go

```diff
@@ -269,12 +270,21 @@ func (pm *PluginManager) scan(pluginDir string, requireSigned bool) error {
 
 		jsonParser := json.NewDecoder(reader)
 
-		// Load the full plugin, and add it to manager
-		if err := loader.Load(jsonParser, plugin, scanner.backendPluginManager); err != nil {
-			return err
+		if signingError != nil {
+			phantomLoader := &PhantomPlugin{*plugin}
```

Comment:
Does this resolve @marefr's concern above, wrt. loading unsigned plugins?

Reply:
I'm not at all familiar with this code but given this:
https://github.com/grafana/grafana/blob/b705329d6a761b6c70bd730d48b3446ea4129e3d/pkg/plugins/plugins.go#L234-L238

is it ever possible this condition to be true?

Reply:
Good eye 😉  You are correct! I just didn't want to remove the phantom loader until I was sure it can be deleted. And now it can thankfully!